server/src/providers/__tests__/diagnostics.backup.ts

```diff
@@ -16,18 +16,18 @@ const createMockDimension = (name: string): LookmlDimension => ({
     type: 'string',
   });
 
-  const createMockMeasure = (name: string): LookmlMeasure => ({
+const createMockMeasure = (name: string): LookmlMeasure => ({
```

Comment:
this test file probably needs to be removed.  i renamed it so its not running right now